package.json

```
@@ -71,13 +71,14 @@
   },
   "scripts": {
     "build": "gatsby build",
-    "check-all": "npm-run-all prettier --parallel lint flow",
-    "ci-check": "npm-run-all prettier:diff --parallel lint flow",
+    "check-all": "npm-run-all prettier --parallel lint flow lint:text",
+    "ci-check": "npm-run-all prettier:diff --parallel lint flow lint:text",
     "dev": "gatsby develop -H 0.0.0.0",
     "flow": "flow",
     "format:source": "prettier --config .prettierrc --write \"{gatsby-*.js,{flow-typed,plugins,src}/**/*.js}\"",
     "format:examples": "prettier --config examples/.prettierrc --write \"examples/**/*.js\"",
     "lint": "eslint .",
```

```suggestion
    "lint": "npm-run-all -p lint:*",
    "lint:eslint": "eslint .",
```

lint コマンドで統一する場合はこんな感じになるかと。

---

一旦これは今度やりますか